Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.mm

```diff
@@ -423,6 +423,7 @@
     // FIXME: This should use a counted replacement for setLayerTreeStateIsFrozen, but
     // the callers of that function are not strictly paired.
 
+    const auto wasWaitingForBackingStoreSwap = m_waitingForBackingStoreSwap;
```

Comment:
I would combine this and the next line into
```
auto wasWaitingForBackingStoreSwap = std::exchange(m_waitingForBackingStoreSwap, false);
```

and we rarely use `const` on local variables.